schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -180,6 +180,8 @@
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-30-preview/Microsoft.Compute.json#/resourceDefinitions/virtualMachines"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-30-preview/Microsoft.Compute.json#/resourceDefinitions/virtualMachineScaleSets"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-30-preview/Microsoft.Compute.json#/resourceDefinitions/extensions"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2017-03-01-preview/Microsoft.Insights.ActionGroup.json#/resourceDefinitions/actiongroups" },
```

Comment:
should be "actionGroups" with capital "G". same issue in 2015-01-01/deploymentTemplate.json.